packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -315,20 +315,28 @@ class _CalendarDatePickerState extends State<CalendarDatePicker> {
     assert(debugCheckHasMaterial(context));
     assert(debugCheckHasMaterialLocalizations(context));
     assert(debugCheckHasDirectionality(context));
+    const double fontSizeToScale = 14.0;
```

Comment:
Where does this int come from? Should we get this value from DefaultTextStyle instead?